.github/workflows/nominatim-docker-test.yml

```diff
@@ -0,0 +1,35 @@
+name: nominatim-docker-test
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  # pull_request:
+  #   branches:
+  #       - master
+  push:
+    branches:
+        - nominatim
+
+  # schedule:
+  #   # Run every Sunday at 4:05 am
+  #   - cron:  '5 4 * * 0'
```

Comment:
Ignore above comment :)

Reply:
I had read somewhere that tests on a schedule can't be run on forked branches, so that's why it was commented out. I'll un-comment it once everything else is ready.